docs/data/material/design-resources/material-ui-sync/material-ui-sync.md

```diff
@@ -56,13 +56,12 @@ After you've added your custom tokens, click on **Regenerate theme** to include
 
 ## Customizing components
 
-The Sync plugin can also generate theme styles for customized components, enabling you to completely change their look and feel and create your custom design system from within Figma.
-
-:::info
-This feature is currently limited to the Button, Switch, and Typography components.
-Support for more components is coming soon.
+:::warning
```

Comment:
I get the idea of making this clearer using the warning callout; however, it feels a bit off to have two warning callouts so close to each other. Should we consider making one of them an info callout instead?

<img width="863" alt="Screenshot 2024-09-02 at 10 12 08" src="https://github.com/user-attachments/assets/b6976a55-5031-46bb-a033-a9f999bdf0e6">

Reply:
It doesn't look that great, but can we do something about it? I'm not really seeing how.